### DISCUSSION TITLE
Content editable -- Table

### DISCUSSION THREAD

**Discussion by acv-98** [ueberdosis/tiptap#1543] [General Questions]
Hi, is it possible to make an entire table (or all of it's cells at once) not editable ?

**Comment by cadars** [comment]
There is `allowTableNodeSelection: false` available as a general setting, would that work?
https://github.com/ueberdosis/tiptap/blob/a640bb15020800c860f22cb52093f7e272f38904/docs/src/docPages/api/nodes/table.md

**Reply by acv-98** [reply]
I managed to solve my problem by making individual table cells disabled with a custom boolean attribute instead + the `contenteditable` setting
